app/controllers/visualizations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class VisualizationsController < ApplicationController
+  def index
+    
+  end
+
+  def show
+  end
+
+  def new
+    @visualization = Visualization.new
+    2.times { @visualization.variables.build }
```

Comment:
Why do we build this twice?

Reply:
For right now, I just hard-coded how many Variables there are per Visualization. So building it twice gives each new Visualization two Variables. I wasn't sure how to make it dynamic yet (I think it would involve adding a small amount of JS), so I was going to save that for later.

Reply:
Okay thanks, this makes perfect sense !